config/docker/kernelci/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2021, 2022 Collabora Limited
```

Comment:
Is it 2021?

Reply:
I should have mentioned that this Dockerfile was based on the one used by kernelci-pipeline which was written in 2021.  I'll add that in the commit message.